catalog/installWebSocket.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# use the command line interface to install websocket package.
+#
+: ${WHISK_SYSTEM_AUTH:?"WHISK_SYSTEM_AUTH must be set and non-empty"}
+AUTH_KEY=$WHISK_SYSTEM_AUTH
+
+SCRIPTDIR="$(cd $(dirname "$0")/ && pwd)"
+CATALOG_HOME=$SCRIPTDIR
+source "$CATALOG_HOME/util.sh"
+
+echo Installing WebSocket package.
+
+createPackage websocket \
+    -a description "Utilities for communicating with WebSockets"
```

Comment:
@starpit The "websocket" package only has one action named "send". I think the description of the send action (below) makes it pretty clear that it is intended for talking to a websocket server (as opposed to acting as one). The only potentially ambiguous text I could find is the description on the line above. I intentionally left it generic to allow us to add more actions in the future, but if you have a suggestion for making this more clear, we can certainly consider that.

Apart from that, I can't see how naming the package anything but "websocket" could be an improvement, as this term accurately describes the intent of the package.